cmd/skopeo/utils.go

```diff
@@ -1,30 +1,72 @@
 package main
 
 import (
+	"errors"
+	"strings"
+
 	"github.com/containers/image/transports"
 	"github.com/containers/image/types"
 	"github.com/urfave/cli"
 )
 
-// contextFromGlobalOptions returns a types.SystemContext depending on c.
-func contextFromGlobalOptions(c *cli.Context) *types.SystemContext {
-	tlsVerify := c.GlobalBoolT("tls-verify")
+func contextFromGlobalOptions(c *cli.Context, credsFlag string) (*types.SystemContext, error) {
+	ctx := newBaseSystemContext(c)
```

Comment:
I don’t think we need the separate `newBaseSystemContext` function now that `credsFlag` is handled inline.  (Eventually we may support `credsFlag == ""` to turn that off, but we don’t need that now.)

This does work just fine, feel free to merge as is.